KeeAgent/ExtensionMethods.cs

```diff
@@ -193,16 +193,16 @@ public static SshPrivateKey GetSshPrivateKey(this PwEntry entry)
               throw new NoAttachmentException();
             };
 
-            SshPublicKey publicKey = null;
+            var privateKey = SshPrivateKey.Read(new MemoryStream(attachment.ReadData()));

```

Comment:
I suspect this will break the older private keys that don't include the public key in the private key data, like https://github.com/dlech/SshAgentLib/blob/master/SshAgentLibTests/Resources/OpenSshTestData/dsa_1.

Reply:
I think you are right, this would throw a `PublicKeyRequiredException` for those keys.